x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -127,30 +127,33 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
   }, []);
 
   const groupTableItems = useMemo(() => {
-    const tableItems = data.changePointsGroups.map(({ group, docCount, pValue }, index) => {
-      const sortedGroup = group.sort((a, b) =>
-        a.fieldName > b.fieldName ? 1 : b.fieldName > a.fieldName ? -1 : 0
-      );
-      const dedupedGroup: Record<string, any> = {};
-      const repeatedValues: Record<string, any> = {};
+    const tableItems = data.changePointsGroups.map(
+      ({ group, docCount, histogram, pValue }, index) => {
+        const sortedGroup = group.sort((a, b) =>
+          a.fieldName > b.fieldName ? 1 : b.fieldName > a.fieldName ? -1 : 0
+        );
+        const dedupedGroup: Record<string, any> = {};
+        const repeatedValues: Record<string, any> = {};
 
-      sortedGroup.forEach((pair) => {
-        const { fieldName, fieldValue } = pair;
-        if (pair.duplicate === false) {
-          dedupedGroup[fieldName] = fieldValue;
-        } else {
-          repeatedValues[fieldName] = fieldValue;
-        }
-      });
+        sortedGroup.forEach((pair) => {
+          const { fieldName, fieldValue } = pair;
+          if (pair.duplicate === false) {
+            dedupedGroup[fieldName] = fieldValue;
+          } else {
+            repeatedValues[fieldName] = fieldValue;
+          }
+        });
 
-      return {
-        id: index,
-        docCount,
-        pValue,
-        group: dedupedGroup,
-        repeatedValues,
-      };
-    });
+        return {
+          id: index,
```

Comment:
Looks like the `id` here can now be set to the group's id coming on the backend - this _should_ fix the expanded row bug - I can fix this in a follow up so this can just go in but wanted to point it out 👍 Thanks, @walterra